api/src/main/java/org/pmiops/workbench/dataset/DataSetMapper.java

```diff
@@ -0,0 +1,15 @@
+package org.pmiops.workbench.dataset;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.db.model.DataDictionaryEntry;
+import org.pmiops.workbench.utils.CommonMappers;
+
+@Mapper(
+    componentModel = "spring",
+    uses = {CommonMappers.class})
+public interface DataSetMapper {
+
+  @Mapping(target = "cdrVersionId", source = "dbModel.cdrVersion.cdrVersionId")
```

Comment:
Is it possible to fully qualify both the source and the target? That might make the line easier to scan in Find results, etc.

Reply:
What would that look like? I'm not sure if there's something we can preprend to `target = cdrVersionId`.